be/CMakeLists.txt

```diff
@@ -588,31 +581,41 @@ if ("${CMAKE_BUILD_TYPE}" STREQUAL "BCC")
     )
 endif()
 
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+    set(ASAN_LIBS -static-libasan)
+    set(LSAN_LIBS -static-liblsan)
+    set(UBSAN_LIBS -static-libubsan tcmalloc)
+    set(TSAN_LIBS -static-libtsan)
```

Comment:
I am not sure which lib of `asan, tsan,lasn,usan` will be used, if the license of that lib is LGPL we may only allow to link the dynamic lib

Reply:
>  we may only allow to link the dynamic lib

That's not the case. We already provide everything that allow the user to relink the application. Even our toolchain is open sourced :)

Reply:
To be more specific, there is nothing in LGPL that mandates dynamic linking/loading. The requirement is that the end user should be able to relink the application to a substitute set of libraries if he so chooses.